src/main/kotlin/utils/util.kt

```diff
@@ -7,10 +7,11 @@ import java.util.*
 
 
 /**
-* init fonts here
-*/
-val headingFont = FontFamily(Font("font/Lato-Bold.ttf"))
-val bodyFont = FontFamily(Font("font/Lato-Regular.ttf"))
+ * init fonts here
+ */
+//changed to google sans
+val headingFont = FontFamily(Font("font/GoogleSans-Bold.ttf"))
```

Comment:
GoogleSans is proprietary, Would need to add license. Lets stick to Lato Font alone as a brand font for KMP-Winget